CircleBar/Classes/SHCircleBar.swift

```diff
@@ -83,8 +89,12 @@ import UIKit
         bezPath.append(UIBezierPath(rect: self.bounds))
         return bezPath
     }
-    func hapticFeedback(){
-        let generator = UINotificationFeedbackGenerator()
-        generator.notificationOccurred(.success)
+    
+    func hapticFeedback(status: Bool){
```

Comment:
this function does not need a boolean as a parameter, it should just do the implementation about haptic feedback
instead of checking for `status` here, please do this conditional where this function is getting called